assignments/4. operators/conditional-operator.md

```diff
@@ -1,62 +1,166 @@
-    * [ ] Mul
-    * [ ] Div
+⛑ Rule
+_ [ ] While substracting and dividing keep in mind the number one should be greater then number two. If not show alert saying `Number Two is larger then Number one`.
+⚡️ Operations
+_ [ ] Add
+_ [ ] Sub
+_ [ ] Mul \* [ ] Div
+
+var a = Number(prompt("Enter No."));
+var b = Number(prompt("Enter Second No."));
+var operator = prompt("Enter add(+)/sub(-)/div(/)/mul(\*)");
+
+if (operator == "add" || operator == "+")
+alert(a+b);
+else if (operator == "sub" || operator == "-"){
+if (a>=b){
+alert(a-b);
+}
+else {
+alert("Number Two is larger then Number one");
+}
+}
+else if (operator == "div" || operator == "/") {
+if (a>b)
+alert(a/b);
+else
+alert("Number Two is larger then Number one");
+}
+else if (operator == "mul" || operator == "*"){
+alert(a*b);
+}
 
 2. 🎖Write a if else statement which checks if the status is single `console.log` the message `John is single` or else `John is married`
+
 ```js
-var firstName = 'John';
-var status = 'single';
+var firstName = "John";
+var status = "single";
 // Your code goes here
+if (status == "single") {
+  console.log("John is Single");
+} else console.log("John is married");
 ```
 
 3. 🎖Write a JavaScript program that takes two `integers` from user (using prompt) and alerts the larger number.
+
 ```js
 // your code goes here
+var a = Number(prompt("Enter No."));
+var b = Number(prompt("Enter other No."));
+if (a > b) alert("larger no. is " + a);
+else alert("larger no. is " + b);
 ```
 
 4. 🎖Write a JavaScript conditional statement to find the sign (+, -) of product of three numbers. Take those three numbers from user using `prompt`. Display an alert box with the specified sign.
 
 ```js
 // Your code goes here
+var a = Number(prompt("Enter No."));
+var b = Number(prompt("Enter other No."));
+var c = Number(prompt("Enter third No."));
+var d = a * b * c;
+if (d > 0) alert("+ve");
+else if (d < 0) alert("-ve");
+else alert("0");
 ```
 
 ## Switch Statement
 
 1. 🎖Using switch statement do the following
 
 Take a number value from user and alert the message if it matches the conditions.
-* [ ] ONE, if `number` is equal to 1.
-* [ ] TWO, if `number` is equal to 2.
-* [ ] THREE, if `number` is equal to 3.
-* [ ] FOUR, if `number` is equal to 4.
-* [ ] FIVE, if `number` is equal to 5.
-* [ ] SIX, if `number` is equal to 6.
-* [ ] SEVEN, if `number` is equal to 7.
-* [ ] EIGHT, if `number` is equal to 8.
-* [ ] NINE, if `number` is equal to 9.
-* [ ] PLEASE TRY AGAIN, if  is none of the above.
+
+- [ ] ONE, if `number` is equal to 1.
+- [ ] TWO, if `number` is equal to 2.
+- [ ] THREE, if `number` is equal to 3.
+- [ ] FOUR, if `number` is equal to 4.
+- [ ] FIVE, if `number` is equal to 5.
+- [ ] SIX, if `number` is equal to 6.
+- [ ] SEVEN, if `number` is equal to 7.
+- [ ] EIGHT, if `number` is equal to 8.
+- [ ] NINE, if `number` is equal to 9.
+- [ ] PLEASE TRY AGAIN, if is none of the above.
+
 ```js
 // Your code goes here
+var a = +prompt("Enter Num");
```

Comment:
Name your variable properly. Name of the variable should be what value it contains.